3.2/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -6,13 +6,13 @@ port = ${CONTAINER_PORT}
 # Default: /var/lib/mongodb/data
 dbpath = ${MONGODB_DATADIR}
 
-# Disable data file preallocation. Default: true
+# Disable data file preallocation.
 # only for mounted data directory from older MongoDB server
-noprealloc = ${MONGODB_NOPREALLOC}
+noprealloc = true
```

Comment:
If we are removing the env var, why do we keep the value in the configuration template?